go/tpm2/apps/QuoteServer/server.go

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2016, Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"log"
+	"net"
+
+	"github.com/jlmucb/cloudproxy/go/tao"
+	"github.com/jlmucb/cloudproxy/go/tpm2"
+	"github.com/jlmucb/cloudproxy/go/util"
+)
+
+var network = flag.String("network", "tcp", "The network to use for connections")
+var addr = flag.String("addr", "localhost:8121", "The address to listen on")
+var pass = flag.String("pass", "xxx", "The password protecting the policy key")
```

Comment:
It looks like simpleexample is using "xxx" too.
But you're right, the scripts in go/run/scripts use BogusPass.
I can change both this and the simpleexample default password to BogusPass.